sunpy/timeseries/sources/eve.py

```diff
@@ -57,35 +57,63 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, axes=None, title="EVE/ESP Level1", **kwargs):
         """
+        Plots the EVE ESP Leve1 timeseries data.
+
         Parameters
         ----------
+        axes : `matplotlib.axes.Axes`
+            The axes on which to plot the TimeSeries. Defaults to current axes.
         title : `str`, optional
             Plot title.
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
+
+        Returns
+        -------
+        axes : `~matplotlib.axes.Axes`
```

Comment:
```suggestion
        `~matplotlib.axes.Axes`
```

Reply:
Is this the format that's followed everywhere?
Won't we need to change it here as well? https://github.com/sunpy/sunpy/blob/3940d01382c1863e3250b5cc7f96531265b29c43/sunpy/timeseries/timeseriesbase.py#L462

Reply:
It is the format that should be used.

Reply:
Alright then, I'll make that change.